test/test_ops.py

```diff
@@ -1109,8 +1096,8 @@ def clone_and_perform_view(input, **kwargs):
                     # a repeat of the above test if output is not complex valued
                     if (out_type(expected_forward)):
                         grad = torch.randn_like(expected_forward)
-                        expected_forward.backward(math_op_physical(grad))
-                        forward_with_mathview.backward(math_op_view(grad))
+                        expected_forward.backward(grad)
+                        forward_with_mathview.backward(math_op_view(math_op_physical(grad)))
```

Comment:
hmm we still want to test gradient computation when `grad_out` has a math bit set

Reply:
This does have the math bit set. `math_op_physical` eagerly computes `t0 = f(grad)` and then `math_op_view` creates a view `t1 = f_view(t0)` and since the math ops are self  inverses `t1` represents the same value as the original `grad` but as a view.